### ISSUE TITLE
Crashed when on media 

### ISSUE THREAD

**Issue by andyxheli** [nextcloud/android#10830]
### ⚠️ Before posting ⚠️

- [X] This is a **bug**, not a question or an enhancement.
- [X] I've [searched for similar issues](https://github.com/nextcloud/android/issues) and didn't find a duplicate.
- [X] I've written a clear and descriptive title for this issue, not just "Bug" or "Crash".
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Steps to reproduce

Clicked on media tab
Took forever to load crashed

### Expected behaviour

Should not be slow and should not lagg 

### Actual behaviour

Low and crash

### Android version

12

### Device brand and model

SM-G996U1 

### Stock or custom OS?

Stock

### Nextcloud android app version

3.22.0

### Nextcloud server version

24.0.5

### Using a reverse proxy?

I don't know

### Cause of error
```java
Exception in thread "main" java.lang.IllegalStateException: Fragment already added: GalleryFragmentBottomSheetDialog{4801d14} (bf8a8611-c104-4c65-89f5-e130b28def83 tag=data)
    at androidx.fragment.app.FragmentStore.addFragment(FragmentStore.java:92)
    at androidx.fragment.app.FragmentManager.addFragment(FragmentManager.java:1481)
    at androidx.fragment.app.BackStackRecord.executeOps(BackStackRecord.java:387)
    at androidx.fragment.app.FragmentManager.executeOps(FragmentManager.java:1965)
    at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:1873)
    at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:1823)
    at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:1760)
    at androidx.fragment.app.FragmentManager$5.run(FragmentManager.java:547)
    at android.os.Handler.handleCallback(Handler.java:938)
    at android.os.Handler.dispatchMessage(Handler.java:99)
    at android.os.Looper.loopOnce(Looper.java:226)
    at android.os.Looper.loop(Looper.java:313)
    at android.app.ActivityThread.main(ActivityThread.java:8751)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:571)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)

```

### App information
* ID: `com.nextcloud.client`
* Version: `30220090`
* Build flavor: `gplay`

### Device information
* Brand: `samsung`
* Device: `t2q`
* Model: `SM-G996U1`
* Id: `SP1A.210812.016`
* Product: `t2quew`

### Firmware
* SDK: `31`
* Release: `12`
* Incremental: `G996U1UES5CVI8`

**Comment by AlvaroBrey** [comment]
Fix is on #10834, although this should only be triggered if double-tapping the menu button, not by itself

**Comment by damien67** [comment]
I has the same issue, cleared cache and data in Android, logged back in, Media is now working.
